node-fs-writefile/cp.js

```diff
@@ -0,0 +1,9 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+try {
+  const fromText = await readFile(process.argv[2], 'utf8');
+  const toText = process.argv[3];
+  await writeFile(toText, `${fromText} \n`);
```

Comment:
This works. However, the code you have written works for text files, but not for binary files. Since you provide the 'utf8' encoding when reading the file, you will only be able to successfully read text files. And you convert the contents of the read file to text in line 6.

To make this work for binary files (like images), you all have to do is remove the 'utf8' encoding and just pass `fromText` directly to `writeFile`!